corehq/apps/reports/tasks.py

```diff
@@ -424,8 +427,12 @@ def _extract_form_attachment_info(form, properties):
         if content_type == 'text/xml':
             continue
         try:
-            question_id = str(
-                '-'.join(find_question_id(form.form_data, attachment_name)))
+            question_id_components = find_question_id(form.form_data, attachment_name)
+            if question_id_components is None:
+                # NOTE: special case until rd-toolkit bug is fixed, search for question_id again
+                # See https://dimagi-dev.atlassian.net/browse/SAAS-11792
+                question_id_components = find_question_id(form.form_data, attachment_name, last_path=True)
```

Comment:
Could this be combined into `find_question_id()`? I don't think there are multiple places that use `find_question_id()`, but presumably every place that does use this, now or in the future, will need to remember to include this second search.

Reply:
Great point. I created somewhat of a wrapper method which may or may not address the issue. Feel free to voice your opinion if you still feel it does not resolve this.